Tasks/Common/kubernetes-common-v2/kubernetesmanifestutility.ts

```diff
@@ -55,20 +76,31 @@ export async function checkPodStatus(kubectl: Kubectl, podName: string): Promise
         }
     }
     podStatus = getPodStatus(kubectl, podName);
+    const environmentUrl = getEnvironmentUrl();
     switch (podStatus.phase) {
         case 'Succeeded':
         case 'Running':
             if (isPodReady(podStatus)) {
                 console.log(`pod/${podName} is successfully rolled out`);
+            } else {
+                kubectl.describe(KubernetesConstants.KubernetesWorkload.pod, podName);
+                if (environmentUrl)
+                    console.log('For more information go to ' + environmentUrl);
             }
             break;
         case 'Pending':
             if (!isPodReady(podStatus)) {
                 tl.warning(`pod/${podName} rollout status check timedout`);
+                kubectl.describe(KubernetesConstants.KubernetesWorkload.pod, podName);
+                if (environmentUrl)
+                    console.log('For more information go to ' + environmentUrl);
```

Comment:
Every message shown to user should come from task.json. 
If it's a debug statement you can skip it. This would end up raising localization errors later on otherwise.

Reply:
Okay. I'll change that